draft-ietf-jsonpath-base.md

```diff
@@ -1456,10 +1456,6 @@ Since bracket notation is more general than dot notation, it is used to construc
 Single quotes are used to delimit string member names. This reduces the number of characters that
 need escaping when Normalized Paths appear as strings (which are delimited with double quotes) in JSON texts.
 
-The syntax of Normalized Paths is restricted so that there is one and only one way of identifying a given node.
```

Comment:
This is mostly a restatement of the first paragraph, so I'm fine with removing this paragraph.